docs/walkthrough/walkthrough-kind.md

```diff
@@ -151,23 +153,17 @@ To access the Dashboard through the ingress controller it is necessary to set up
 
 For the sake of simplicity we will use the freely available [`nip.io`](https://nip.io/) service to construct a URL pointing to our local cluster.
 
-Assuming the following url `http://tekton-dashboard.127.0.0.1.nip.io`, run the following command to create the ingress rule:
```

Comment:
I think it's worth keeping the original content here, and adding a note afterwards with the instructions for doing this with the installer. It shows some of the utility of the installer while giving a little insight into what it's doing and giving an alternative if someone wants to create the ingress manually for some reason.

Some users may find it odd that we're installing the dashboard again just to add an ingress, so explaining that this could all be done in one step is helpful.

Reply:
I re-added the original content and placed the installer instructions inside an additional note. Thanks